<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Techno-311" failures="1" tests="1" name="AuditLog19" time="40.793" errors="0" timestamp="16 Jun 2022 09:45:19 GMT">
  <testcase classname="io.mosip.testscripts.AuditValidator" name="" time="21.7">
    <failure type="io.mosip.admin.fw.util.AdminTestException" message="Failed at output validation">
      <![CDATA[io.mosip.admin.fw.util.AdminTestException: Failed at output validation
	at io.mosip.testscripts.AuditValidator.test(AuditValidator.java:90)
	at io.mosip.testrunner.MosipTestRunner.startTestRunner(MosipTestRunner.java:93)
	at io.mosip.testrunner.MosipTestRunner.main(MosipTestRunner.java:49)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!--  -->
</testsuite> <!-- AuditLog19 -->
